spec/slim_lint/linter/tab_spec.rb

```diff
@@ -0,0 +1,24 @@
+require 'spec_helper'
+
+describe SlimLint::Linter::Tab do
+  include_context 'linter'
+
+  context 'when a file contains tabs' do
+    let(:slim) { <<-SLIM }
+      .container
+      	p Hello World
```

Comment:
You should use the tab escape sequence (`\t`) to represent the tabs here instead of literal tab characters, as some editors (including mine) will automatically convert tabs to spaces, breaking this test the moment someone else edits this file.

It has the added benefit of making it clearer which character is being rendered (there are many kinds of whitespace characters).